lib/schemas/currencies.json

```diff
@@ -0,0 +1,304 @@
+[
```

Comment:
Ideally, this should be its own module. I can see other libraries needing to pull in currency codes. It is fine here for now.